docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -174,6 +177,92 @@ For more details on getting started with the Synthetics Recorder, see <<syntheti
 
 image::images/synthetics-create-test-script-recorder.png[Elastic Synthetics Recorder after recording a journey and clicking Export]
 
+[discrete]
+[[synthetics-request-param]]
+== Make API requests
+
+You can use the `request` parameter to make API requests independently of the browser interactions.
+
+// Link to Playwright docs?
+// https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests
```

Comment:
Should we link to the Playwright docs here?

Reply:
This part of the docs is confusing it tells how user would use `playwright.request` to perform HTTP requests.